### PR TITLE
A Vote for CabinfeverB as TiDB Reviewer

### PR DESCRIPTION
**Pull request by qw4990**
# A Vote for CabinfeverB as TiDB Reviewer

## Proposal

[@CabinfeverB](https://github.com/CabinfeverB) has been working on the related repositories like 'tikv/pd' for more than one year. ​He fixed some bugs related to TSO in the pd client to optimize recovery time. In recent months, he has been working on the feature `Resource Control` in TiDB. He was involved in resource group bug fixes, dynamic calibrate resource, runaway query management, etc.

* [Authored commits](https://github.com/pingcap/tidb/commits?author=CabinfeverB)

I (@qw4990) hereby nominate @CabinfeverB as TiDB Reviewer and call for a vote.

## Deadline

The vote will be open for at least 3 days unless there is an objection or not enough votes.

## Scope

* Team TiDB

## Result

See also: https://github.com/pingcap/community/pull/769

### CODE REVIEW

**Review by nolouch — COMMENTED** [review]
+1

File: teams/tidb/membership.json [hunk 1]
```diff
@@ -98,7 +98,7 @@
         "ChenPeng2013",
         "Defined2014",
         "dhysum",
-	"elsa0520",
+	    "elsa0520",
```
Comment (disksing) [inline teams/tidb/membership.json:101]: cleanup?

File: teams/tidb/membership.json [hunk 2]
```diff
@@ -116,6 +116,7 @@
         "longfangsong",
         "mengxin9014",
         "mornyx",
+        "CabinfeverB",
```
Comment (bb7133) [inline teams/tidb/membership.json:119]: Please follow the the alphabetical(and case) order, move this betwen "BornChanger" and "ChenPeng2013"

File: votes/0764-CabinfeverB-as-tidb-reviewer.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# A Vote for CabinfeverB as TiDB Reviewer
```
Comment (bb7133) [inline votes/0764-CabinfeverB-as-tidb-reviewer.md:1]: Please fix the file name to match PR ID('0770').

File: votes/0764-CabinfeverB-as-tidb-reviewer.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+## Proposal
+
+[@CabinfeverB](https://github.com/CabinfeverB) has been working on the related repositories like 'tikv/pd' for more than one year. ​He fixed some bugs related to TSO in the pd client to optimize recovery time. In recent months, he has been working on the feature `Resource Control` in TiDB. He was involved in resource group bug fixes, dynamic calibrate resource, runaway query management, etc.
```
Comment (bb7133) [inline votes/0764-CabinfeverB-as-tidb-reviewer.md:5]:
```suggestion
[@CabinfeverB](https://github.com/CabinfeverB) has been working on the related repositories like 'pingcap/tidb' for more than one year. ​He fixed some bugs related to TSO in the pd client to optimize recovery time. In recent months, he has been working on the feature `Resource Control` in TiDB. He was involved in resource group bug fixes, dynamic calibrate resource, runaway query management, etc.
```

**Comment by bb7133** [comment]
I will make a '+1' vote for the nomination.

Please address my comment.

**Comment by qw4990** [comment]
All updated @bb7133  PTAL

File: teams/tidb/membership.json (hunk 1 above)
Comment (CabinfeverB) [reply teams/tidb/membership.json:102]: It seems be caused by using `\t` by PR762

File: teams/tidb/membership.json [hunk 5]
```diff
@@ -96,9 +96,10 @@
         "Benjamin2037",
         "BornChanger",
         "ChenPeng2013",
+        "CabinfeverB",
         "Defined2014",
         "dhysum",
-	"elsa0520",
+	    "elsa0520",
```
Comment (qw4990) [inline teams/tidb/membership.json:102]:
```suggestion
	"elsa0520",
```

**Review by disksing — APPROVED** [review]

**Pull request merged**